pandas/conftest.py

```diff
@@ -591,6 +593,91 @@ def narrow_series(request):
     return _narrow_series[request.param].copy()
 
 
+_dtype_nuisance_arr_map = {
+    "float64": [
+        -np.inf,
+        -50,
+        -1,
+        -1e-20,
+        -1e-25,
+        -1e-50,
+        0,
+        1e-40,
+        1e-20,
+        1e-10,
+        2,
+        40,
+        np.inf,
+    ],
+    "float32": [
+        -np.inf,
+        -50,
+        -1,
+        -1e-20,
+        -1e-25,
+        -1e-45,
+        0,
+        1e-40,
+        1e-20,
+        1e-10,
+        2,
+        40,
+        np.inf,
+    ],
+    "uint8": [np.iinfo(np.uint8).min, 1, 2, 100, np.iinfo(np.uint8).max],
+    "int64": [
+        np.iinfo(np.int64).min,
+        -100,
+        0,
+        1,
+        9999,
+        100000,
+        1e10,
+        np.iinfo(np.int64).max,
+    ],
+    "object": [NegInfinity(), "1", "A", "BA", "Ba", "C", Infinity()],
+}
+
+_dtype_na_map = {
```

Comment:
can you put this *inside* the fixture, this is generally a special case

Reply:
Done